cotel/.idea/modules/cotel.iml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.... fuck @Cotel uploading `.idea` ... You are a beginner or lion? beginner or lion!!!?